### PR TITLE
Network february release network interface json update

### PR DESCRIPTION
**Pull request by bhbhise**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] Other, please clarify  - Adding a new property to NetworkworkInterface resource.

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by kairu-ms — APPROVED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/networkInterface.json [hunk 1]
```diff
@@ -1160,6 +1160,11 @@
           "readOnly": true,
           "$ref": "./network.json#/definitions/ProvisioningState",
           "description": "The provisioning state of the network interface resource."
+        },
+        "workloadType": {
+          "readOnly": false,
```
Comment (kairu-ms) [inline specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/networkInterface.json:1165]: You don't need to add this property if readOnly is false.
```suggestion
```

File: specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/networkInterface.json [hunk 2]
```diff
@@ -1160,6 +1160,11 @@
           "readOnly": true,
           "$ref": "./network.json#/definitions/ProvisioningState",
           "description": "The provisioning state of the network interface resource."
+        },
+        "workloadType": {
```
Comment (kairu-ms) [inline specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/networkInterface.json:1164]: Does the value of property is enumerable?

**Review by kairu-ms — CHANGES_REQUESTED** [review]

**Comment by kairu-ms** [comment]
@msyyc Python SDK Track2 is failed. Please have a look.

File: specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/networkInterface.json (hunk 2 above)
Comment (bhbhise) [reply specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/networkInterface.json:1164]: It will be a string

**Comment by kairu-ms** [comment]
Please fix Cross Version Breaking Change issue. Thanks.

**Review by kairu-ms — APPROVED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/networkInterface.json (hunk 1 above)
Comment (bhbhise) [reply specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/networkInterface.json]: removed this.

**Comment by kairu-ms** [comment]
Approve Cross Version BreakingChange, because it is not cause by this PR.

**Pull request merged**